### PR TITLE
Add type definition of the module format-number

### PR DESCRIPTION
**Pull request by codelovesme**
Add type definition of the module format-number

Amend

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by plantain-00 — CHANGES_REQUESTED** [review]
The package has a type declaration, so this PR seems not necessary: https://github.com/componitable/format-number/blob/master/index.d.ts

**Comment by codelovesme** [comment]
> The package has a type declaration, so this PR seems not necessary: https://github.com/componitable/format-number/blob/master/index.d.ts

For version 3 it has, yes.
However,
There is no type declaration for version 2.0.
Declarations I sent is for version 2.0

**Review by plantain-00 — APPROVED** [review]

File: types/format-number/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Type definitions for npm package [format-number](https://www.npmjs.com/package/format-number) version 2.0
```
Comment (sheetalkamat) [inline types/format-number/README.md]: Don't need this file. The version is detected from headers in index.d.ts

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

**Comment by codelovesme** [comment]
I have removed the file. Is there any glitch ? @sheetalkama

**Pull request merged**